x-pack/plugins/security_solution/server/endpoint/routes/actions/audit_log.ts

```diff
@@ -15,6 +15,9 @@ import { withEndpointAuthz } from '../with_endpoint_authz';
 
 /**
  * Registers the endpoint activity_log route
+ * @deprecated
+ * @removeBy 9.0.0
```

Comment:
Do we wish to allow this to work until the last release before `9.0.0`? Shouldn't this be `8.7.0` instead, as in, it should work in `8.7` but not in `8.8`?

Reply:
Ideally, I'd love to delete it sooner but I believe we aren't supposed to break APIs outside of major releases.